Apps/WebClient/src/ClientApp/app/components/timeline/comment.vue

```diff
@@ -108,24 +130,22 @@
             ></b-form-textarea>
           </b-form>
         </div>
-        <div class="d-flex comment-button px-3 flex-row">
+        <div class="d-flex comment-button-wrapper flex-row">
```

Comment:
How about just using columns? Those ones resize and position themselves depending on the size of the availiable space